src/Cmd/Extra.elm

```diff
@@ -89,18 +81,18 @@ attempt =
 
 -}
 maybe : Maybe msg -> Cmd msg
-maybe maybe =
-    Maybe.map perform maybe
+maybe maybeMsg =
+    Maybe.map perform maybeMsg
         |> Maybe.withDefault Cmd.none
 
 
 {-| Construct from Result.
 
-    resultCmd : Cmd String
+    resultCmd : Cmd (Result Never Int)
     resultCmd =
-      fromResult toString (Ok 1)
+      fromResult identity (Ok 1)
```

Comment:
I'm not sure about those functions - they are not covered with elm-verify-examples, only "bad" code is (-> Cmd.none) Not sure what to do with them